client/config/crd/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - snapshot.storage.k8s.io_volumesnapshotclasses.yaml
+  - snapshot.storage.k8s.io_volumesnapshotcontents.yaml
+  - snapshot.storage.k8s.io_volumesnapshots.yaml
```

Comment:
@pohly Do you see any issue with this change?  We install snapshot CRDs from external-snapshotter/pkg/client/config/crd folder:
https://github.com/kubernetes-csi/csi-release-tools/blob/master/prow.sh#L742

Reply:
Adding the kustomization.yaml makes sense to me, I can imagine that some users may want to use the directory as their base.

But what does that have to do with prow.sh? Our usage of the files will simply ignore the additional kustomization.yaml.

Reply:
Just want to double check.  It should be fine.

Reply:
@itspngu Can you update the "Install Snapshot CRDs:" section from "kubectl create -f client/config/crd" to install individual CRDs?  I want to avoid confusion for folks who are new to this.
https://github.com/kubernetes-csi/external-snapshotter/blob/master/README.md#usage

Reply:
> install individual CRDs

Good catch! Are you sure you want all the individual files though? That'd be rather verbose. Part of the beauty of kustomize is that you can easily do something like this:

```
$ kubectl kustomize client/config/crd | kubectl create -f - --dry-run=server
Error from server (AlreadyExists): error when creating "STDIN": customresourcedefinitions.apiextensions.k8s.io "volumesnapshotclasses.snapshot.storage.k8s.io" already exists
Error from server (AlreadyExists): error when creating "STDIN": customresourcedefinitions.apiextensions.k8s.io "volumesnapshotcontents.snapshot.storage.k8s.io" already exists
Error from server (AlreadyExists): error when creating "STDIN": customresourcedefinitions.apiextensions.k8s.io "volumesnapshots.snapshot.storage.k8s.io" already exists
```

As such, I'd update the readme with something similar to the above where needed, unless you prefer the explicit way using individual files.

Reply:
That should be fine.  Thanks

Reply:
I pushed an update to the readme. Let me know if there's anything else to change.